Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "Plots"
 uuid = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 author = ["Tom Breloff (@tbreloff)"]
-version = "1.35.0"
+version = "1.35.1"
```

Comment:
We usually prefer to push a dedicated commit with the ver num in the commit text for version bump (this help visually inspecting commits).

Reply:
oh, okay. I'll do some force push magic once CI is happy.

Reply:
If you do it now, you'll spare waiting for CI twice ;)

Reply:
The policy here in `Plots` is to squash `PRs` before merging, again for clarity.
So what I meant was remove the version bump in this `PR`, merge when `CI` is green, and then edit `Project.toml` in the github interface to only increase the version, with the version in the title commit as e.g. https://github.com/JuliaPlots/Plots.jl/commit/87d2470df3203e378f43c6c2a877789c2d0e2a24.
I wasn't clear enough on that one (you'll know for the next `PR`, no need to change it now).